docs/application/native/tutorials/details/app-filtering.md

```diff
@@ -323,6 +321,10 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/vision.qrcode_generation` | Specify this key, if the application requires the QR code generation feature. | 2.2.1 |
 | `http://tizen.org/feature/vision.qrcode_recognition` | Specify this key, if the application requires the QR code recognition feature. | 2.2.1 |
 
+If you want to verify which features are necessary for using a
+specific API, see the related feature in the native [API
+Reference](../../../../org.tizen.native.mobile.apireference/index.html).
```

Comment:
Sorry for the late review @SunmLee,
the link to the native api files should be `../../api/overview.md` in this file.

Reply:
@mijin85cho Thanks for review.
It will be applied.